calibrators/docs/tutorials/marker_radar_lidar_calibrator.md

```diff
@@ -0,0 +1,234 @@
+If it does not become available, it means that either the required `tf` or services are not available. In this case, since the `tf` are published by the provided rosbag, run the bag with the command `ros2 bag play radar_lidar.db3 --clock -r 0.5` and click the `Calibrate` button.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/menu3.jpg" alt="menu3">
+</p>
+
+### Extract background model
+
+Once the user starts running the tutorial rosbag, the pointcloud should appear in `rviz` as shown in the example below. Press the `Extract Background Model button` in the UI to start extracting the background.
+
+Note that the user should put the radar reflector out of the calibration area before clicking the button. Ideally, there shouldn't be any moving objects like humans and vehicles in the calibration area.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/rviz1.jpg" alt="rviz1" width="500">
+</p>
+
+Once the user clicks the button, it will show like the image below.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/rviz2.jpg" alt="rviz2" width="500">
+</p>
+
+Once the background is extracted, it will show like the image below. The user can see that the `Add lidar-radar pair` button is enabled.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/rviz3.jpg" alt="rviz3" width="500">
+</p>
+
+Also, the following text should be shown in the console.
+
+```text
+[marker_radar_lidar_calibrator]: Finished background model initialization
+```
+
+Additionally, the user can click the options: `lidar_background_pointcloud`, `lidar_foreground_pointcloud`, and `radar_background_pointcloud` on the left bar of the `rviz` for visualizing the output of background and foreground pointcloud. As described in the [documentation](../../README.md), the `radar_reflections` is the same as the `radar_foreground_pointcloud`, which is visible as default.
+
+<table>
+  <tr>
+    <td><img src="../images/marker_radar_lidar_calibrator/lidar_background.jpg" alt="lidar_background" width = 700px height = 300px ></td>
+    <td><img src="../images/marker_radar_lidar_calibrator/lidar_foreground.jpg" alt="lidar_foreground" width = 700px height = 300px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Lidar background points.</p></td>
+    <td><p style="text-align: center;">Lidar foreground points.</p></td>
+  </tr>
+</table>
+
+<table>
+  <tr>
+    <td><img src="../images/marker_radar_lidar_calibrator/radar_background.jpg" alt="radar_background" width = 700px height = 300px></td>
+    <td><img src="../images/marker_radar_lidar_calibrator/radar_foreground.jpg" alt="radar_foreground" width = 700px height = 300px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Radar background points.</p></td>
+    <td><p style="text-align: center;">Radar foreground points.</p></td>
```

Comment:
The radar foreground figure displays two types of data. Please add an explanation or labels in the caption.
Somewhere it should be explained that why the markers have they shape they have (there is no elevation, so Z is unknown)

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/0e04bea19c616ad98987600bc23cc3587e12a29a and https://github.com/tier4/CalibrationTools/pull/164/commits/7f4b0218728545784f3818b9288e8a58856e2968

Reply:
The proposed text is contradictory:
`Radar foreground points (purple line).`
point != line

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/7352fd302bf33c1aafc843f2c3738ddf215aef5d

Reply:
The lower left caption is completely wrong

Reply:
Could you tell me what is wrong?

Reply:
![image](https://github.com/tier4/CalibrationTools/assets/4712270/ddaa8f03-09f3-4194-83ad-f2227e34a969)
What I see here is the raw lidar pointcloud (in white) and the radar foreground detections, represented as line markers

Reply:
I add additional info to explain https://github.com/tier4/CalibrationTools/pull/164/commits/8802c6cd66465e80e56cefb3a19583790954d41b